README.md

```diff
@@ -162,23 +161,23 @@ For EMU_TN you may add the integrator timestep in nanoseconds, the numerical acc
 ```python
 # replace the corresponding section in the above code example with this to
 # add further configuration
-from sdk.device.device_types import DeviceType
+from sdk.device.emulator_types import EmulatorType
 from sdk.device.configuration import EmuTNConfig
 
 configuration = EmuTNConfig(dt = 10.0, precision = "normal", max_bond_dim = 100)
-batch = sdk.create_batch(serialized_sequence, [job1,job2], device_type=DeviceType.EMU_TN, configuration=configuration)
+batch = sdk.create_batch(serialized_sequence, [job1,job2], emulator=EmulatorType.EMU_TN, configuration=configuration)
 ```
 
 For EMU_FREE you may add some default SPAM noise. Beware this makes your job take much longer.
 
 ```python
 # replace the corresponding section in the above code example with this to
 # add further configuration
-from sdk.device.device_types import DeviceType
+from sdk.device.emulator_types import EmulatorType
 from sdk.device.configuration import EmuFreeConfig
```

Comment:
nit: for convenience for the users, could we export these objects in the `sdk.device` module ?

Reply:
Done